enumerator/zsyscall_windows.go

```diff
@@ -131,8 +131,8 @@ func setupDiOpenDevRegKey(set hDevInfo, devInfo *pspDevInfoData, scope dicsScope
 	return
 }
 
-func setupDiGetDeviceRegistryProperty(set hDevInfo, devInfo *pspDevInfoData, property deviceProperty, propertyType *uint32, outValue *byte, outSize *uint32, reqSize *uint32) (res bool) {
```

Comment:
This file is autogenerated by `go generate` from `syscall_windows.go` file.
Please change macro in `syscall_windows.go` and after just run `go generate`

Reply:
Thanks for your notice, my _pull requests_ have been updated.